scripts/dashboard/install-cephadm-e2e-deps.sh

```diff
@@ -86,5 +86,7 @@ sudo chmod +x /usr/local/bin/kcli
 
 # KCLI cleanup function can be found here: https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/ci/cephadm/start-cluster.sh
 sudo mkdir -p /var/lib/libvirt/images/ceph-dashboard
+kcli delete plan ceph -y || true
+kcli delete network ceph-dashboard -y
```

Comment:
Aren't we performing this clean-up after executing the job? It might not make sense in a local run, but in Jenkins we should leave the node as it was before starting the job (ok, perhaps we can leave the VM disk images stored in the cache to avoid downloading them on every run).

Reply:
We remove the plan after every job but we didn't removed the network in the [cleanup](https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/ci/cephadm/start-cluster.sh#L9). I added this today in [this PR](https://github.com/ceph/ceph/pull/44825/files). But since most of the machine has the network already its best to recreate the network before starting the test for now.

Reply:
ok, make sense!